manifest.json

```diff
@@ -2,27 +2,33 @@
   "manifest_version": 2,
   "name": "Custom Elements Locator",
   "description": "This extension will find any custom element on a page.",
-  "version": "1.3",
+  "version": "1.4.0",
```

Comment:
This PR doesnt actually change the extension itself, right? Is a version bump needed in this case, since this PR wont lead to a new publish of the extension itself?